src/omnicore/doc/release-notes.md

```diff
@@ -1,7 +1,7 @@
-Omni Core v0.0.11
-=================
+Omni Core v0.0.11.1
+===================
 
-v0.0.11 is a major release and consensus critical in terms of the Omni Layer protocol rules. An upgrade is mandatory, and highly recommended. Prior releases will not be compatible with new behavior in this release.
+v0.0.11.1 is a major release and consensus critical in terms of the Omni Layer protocol rules. An upgrade is mandatory, and highly recommended. Prior releases will not be compatible with new behavior in this release.
 
```

Comment:
0.0.11.1 is not consensus critical (compared to 0.0.11), though I'm not sure that really counts since not many folks will be on 0.0.11.

How about:

> v.0.0.11.1 is a bugfix release which resolves a critical bug in the RPC API whereby under certain circumstances retrieving data about a sell offer may trigger a math failsafe and cause the automatic shutdown of the client.

You could then include the original v0.0.11 "v0.0.11 is a major release and consensus critical in terms..." underneath perhaps?

Just thinking out loud.